src/dsp_tools/resources/start-stack/start-stack-config.yml

```diff
@@ -1,4 +1,4 @@
 ---
 
 # take commit hash of latest DSP-API release from https://github.com/dasch-swiss/dsp-api/commits/main
-DSP-API commit: c95b146bcc75828b84619b9e158d10353934fcab
+DSP-API commit: f124f855043443d355a6d07b6fb384361c7b4e24
```

Comment:
This is the latest commit of DSP-API, but not the latest release. What we need is the commit hash of the commit "chore(main): release 30.0.0"

Reply:
alles klar danke